pkg/controller/user/export.go

```diff
@@ -51,17 +59,42 @@ func (c *Controller) HandleExport() http.Handler {
 			return
 		}
 
-		w.Header().Add("Content-Disposition", "")
-		w.Header().Add("Content-Type", "text/CSV")
+		filename := fmt.Sprintf("%s-users.csv", time.Now().Format(project.RFC3339Squish))
```

Comment:
Yay nice touch.

I did this to the bulk-upload files in javascript: https://github.com/google/exposure-notifications-verification-server/pull/1469/files

Now I'm wondering if I should flip it with the date first (for string sortability)

Reply:
The date is first

Reply:
I agree! I'm saying can you change where I did it wrong for consistency (or I'll slip that in to my next PR)